modules/tests/test/copy_template_test.go

```diff
@@ -61,7 +58,8 @@ var _ = Describe("Copy template task", func() {
 					LimitTestScope: NamespaceTestScope,
 				},
 				TaskData: testconfigs.CopyTemplateTaskData{
-					ForbiddenNamespace: openshiftNamespace,
+					ForbiddenNamespace: SystemTargetNS,
```

Comment:
why not just pass it to the TargetNamespace? In that case we would not need the ForbiddenNamespace variable and it would simplify the code